programs/marginfi/src/instructions/marginfi_account/flashloan.rs

```diff
@@ -144,4 +156,29 @@ pub struct LendingAccountEndFlashloan<'info> {
     pub marginfi_account: AccountLoader<'info, MarginfiAccount>,
     #[account(address = marginfi_account.load()?.authority)]
     pub signer: Signer<'info>,
+    // Note: there is just one FeeState per program, so no further check is required.
+    #[account(
+        seeds = [FEE_STATE_SEED.as_bytes()],
```

Comment:
Nit: we don't need the seed check, this account is unique per-program deployment.